apps/web/src/lang/fr.json

```diff
@@ -6585,6 +6613,10 @@
     "defaultMessage": "Créer une classification",
     "description": "Page title for the classification creation page"
   },
+  "faS0+E": {
+    "defaultMessage": "Traiter l’état",
```

Comment:
I believe this is incorrect. The original string, Process status, is using the word process as a noun (a process) where as _traiter_ is the verb (to process).